tools/pythonpkg/tests/fast/test_type.py

```diff
@@ -75,6 +75,7 @@ def test_struct_type(self):
         type = duckdb.struct_type({'a': BIGINT, 'b': BOOLEAN})
         assert str(type) == 'STRUCT(a BIGINT, b BOOLEAN)'
 
+        # FIXME: create an unnamed struct when fields are provided as a list
```

Comment:
Do you know if this is still the case? I think that we are using default names for the struct fields, as this is not the `ROW` function.

Reply:
```py
import duckdb
from duckdb.value.constant import StructValue

duckdb.execute("""
	create table tbl (a STRUCT(a BIGINT, b VARCHAR, c BOOL))
""")

# Equivalent to ROW(42, 'bla', True)
res = duckdb.execute("insert into tbl select $col_a", {
	'col_a': StructValue((42, 'bla', True), [int, str, bool])
}).fetchall()

res = duckdb.table("tbl").fetchall()

print(res)
```

Currently this passes, because we don't check for name equality, but after this PR this no longer works because the provided struct value is considered "named"